components/datetime/src/provider/time_zones.rs

```diff
@@ -197,3 +197,20 @@ pub struct MetaZoneSpecificNamesShortV1<'data> {
     #[cfg_attr(feature = "serde", serde(borrow))]
     pub overrides: ZeroMap2d<'data, TimeZoneBcp47Id, TinyStr8, str>,
 }
+
+/// An ICU4X mapping to the metazones at a given period.
+/// See CLDR-JSON metaZones.json for more context.
+#[icu_provider::data_struct(MetaZonePeriodV1Marker = "time_zone/metazone_period@1")]
```

Comment:
when I ran `cargo make testdata`, I did not see a line like `[icu_datagen] Writing key: time_zone/metazone_period@1`. 

Did I miss anything?

Reply:
Yes; add the key to `get_all_keys` in registry.rs